### PR TITLE
Re-implement using Python

### PR DESCRIPTION
**Pull request by dwendland**
* Re-development of existing code with Python Flask and gunicorn

### CODE REVIEW

File: api/token.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    # Build Token object
+    try:
+        from api.models.token import Token
+        client_id = request.form.get('client_id')
+        ar_token = Token(
+            eori=client_id,
+            access_token=auth_data['access_token'],
+            expires=int(time.time() * 1000) + (1000*auth_data['expires_in']))
+    except Exception as bex:
+        print(bex)
```
Comment (wistefan) [inline api/token.py:41]: Why print & logger?

**Review by wistefan — APPROVED** [review]

File: api/token.py (hunk 1 above)
Comment (dwendland) [reply api/token.py]: Line is removed

**Comment by tarmo-profi** [comment]
What is the reason for changing JS to python?

**Comment by dwendland** [comment]
> What is the reason for changing JS to python?

Better maintainability and tests compared to the code before, needed a more flexible code base for upcoming DSBA-compliant implementations supporting trusted issuers lists

**Pull request merged**